homeassistant/components/sensor/monzo.py

```diff
@@ -0,0 +1,157 @@
+"""
+Support for Monzo service.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.monzo/
+"""
+import logging
+from datetime import timedelta
+
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import (
+    CONF_NAME, CONF_MONITORED_VARIABLES, CONF_ACCESS_TOKEN)
+from homeassistant.helpers.entity import Entity
+from homeassistant.util import Throttle
+import homeassistant.helpers.config_validation as cv
+
+REQUIREMENTS = ['monzo==0.5.3']
+
+_LOGGER = logging.getLogger(__name__)
+
+CONF_ACCOUNT_ID = 'account_id'
+
+ICON_EUR = 'mdi:currency-eur'
+ICON_GBP = 'mdi:currency-gbp'
+ICON_INR = 'mdi:currency-inr'
+ICON_NGN = 'mdi:currency-ngn'
+ICON_RUB = 'mdi:currency-rub'
+ICON_TRY = 'mdi:currency-try'
+ICON_USD = 'mdi:currency-usd'
+
+DEFAULT_NAME = 'Monzo'
+
+VARIABLE_TYPES = {
+    'balance': 'Balance',
+    'currency': 'Currency',
+    'spend_today': 'Spend today',
+    'local_currency': 'Local currency',
+    'local_exchange_rate': 'Local exchange rate'
+}
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_ACCESS_TOKEN): cv.string,
+    vol.Required(CONF_MONITORED_VARIABLES, default=[]):
+        vol.All(cv.ensure_list, [vol.In(VARIABLE_TYPES)]),
```

Comment:
Why not set the `balance` as default and make it optional? If it's required then no default should be set.